### PR TITLE
DDFFORM 166 accordion

### PR DESCRIPTION
**Pull request by LasseStaus**
- Make the absoluteBaseUrl available to all templates.
- Create paragraph accordion
- Add field_accordion_description for accordion paragraph
- Add field_accordion_title for accordion paragraph
- Added paragraph--accordion.html.twig.

#### Link to issue

[jira link](https://reload.atlassian.net/jira/software/c/projects/DDFFORM/boards/485?selectedIssue=DDFFORM-166)

#### Description

This pr adds a paragraph for an accordion. 

Each accordion is its' own seperate paragraph. 

#### Screenshot of the result
image

#### Additional comments or questions

In novel.theme i added an absoluteBaseUrl. Also specified in the commit. Comment on this please.

### CODE REVIEW

File: web/themes/custom/novel/novel.theme [hunk 1]
```diff
@@ -109,6 +109,7 @@ function novel_preprocess_dpl_react_app__search_result(array &$variables): void
 function novel_preprocess(array &$variables): void {
   $directory = $variables['directory'] ?? 'web/themes/custom/novel';
   $variables['baseIconPath'] = $directory . '/assets/dpl-design-system/icons';
+  $variables['absoluteBaseUrl'] = \Drupal::request()->getSchemeAndHttpHost();
```
Comment (rasben) [inline web/themes/custom/novel/novel.theme:112]: isnt it a bit extreme to add this to **every** preprocess there is?
wouldnt it be better to just add it to e.g. `novel_preprocess_paragraph`

I do know that you're adding this to the existing hook, but atleast that is really simple code (e.g. based on static strings)

File: web/themes/custom/novel/templates/paragraphs/paragraph--accordion.html.twig [hunk 2]
```diff
@@ -0,0 +1,11 @@
+<details class="disclosure disclosure--paragraph-width text-body-large">
```
Comment (rasben) [inline web/themes/custom/novel/templates/paragraphs/paragraph--accordion.html.twig:1]: please remember to use ``{{ attributes.addClass(....) }}`, so we can also control it through future `preprocess`'es

**Review by rasben — CHANGES_REQUESTED** [review]

File: web/themes/custom/novel/templates/paragraphs/paragraph--accordion.html.twig (hunk 2 above)
Comment (LasseStaus) [reply web/themes/custom/novel/templates/paragraphs/paragraph--accordion.html.twig:1]: Good point, will do!

File: web/themes/custom/novel/novel.theme (hunk 1 above)
Comment (LasseStaus) [reply web/themes/custom/novel/novel.theme:112]: I'm fine with moving it to paragraph preprocess instead 👍

**Pull request merged**